### PR TITLE
Add PHP 8.0 images

### PR DESCRIPTION
**Pull request by kierenevans**
Some extensions aren't ready at the moment.

Needs Xdebug v3 as v2 isn't compatible.

Also convert from Travis to Jenkins.

### CODE REVIEW

**Comment by kierenevans** [comment]
Fails on building the event extension.

File: installer/stretch/precompile.sh [hunk 1]
```diff
@@ -10,16 +10,21 @@ source ./lib/functions.sh
 
 function main()
 {
+    local KEEPALIVE_PID
     for extension in extensions/*
     do
         extension_name="${extension%.sh}"
         extension_name="${extension_name#extensions/}"
         echo -n "Installing ${extension}..."
+        bash -c 'for i in {0..20..1}; do sleep 30 && echo -n "."; done' &
```
Comment (hgajjar) [inline installer/stretch/precompile.sh:19]: what is the last `..1` doing in `{0..20..1}` range?

Comment (kierenevans) [reply installer/stretch/precompile.sh:19]: Increment. We could probably leave it off :)

Comment (hgajjar) [reply installer/stretch/precompile.sh:19]: 👍  never saw it before :)

**Comment by kierenevans** [comment]
Build passing in Jenkins (build and test) 🥳 
Need to get it pushing on "main" branch still.

**Review by dcole-inviqa — APPROVED** [review]

**Pull request merged**